pmd-kotlin/pom.xml

```diff
@@ -11,11 +11,25 @@
         <relativePath>../</relativePath>
     </parent>
 
+    <properties>
+        <antlr4.visitor>true</antlr4.visitor>
+    </properties>
+
     <build>
+        <!-- needed for pmd sonar plugin -->
```

Comment:
The filtering of resources is needed independent of pmd sonar plugin.
It replaces `${pmd.website.baseurl}` in the ruleset.....

Reply:
I see, so comment can be deleted.

Reply:
I see, so xml comment can be deleted